package-scripts.js

```diff
@@ -10,13 +10,23 @@ module.exports = {
     },
     validate: {
       description: 'Runs code formatting, linting and tests',
-      script: 'nps format lint test'
```

Comment:
Validation shouldn't format, otherwise the precommit hook makes uncommitted formatting changes while still passing, which is not desirable.